src/ng/rootScope.js

```diff
@@ -1133,7 +1133,7 @@ function $RootScopeProvider() {
        *   - `currentScope` - `{Scope}`: the scope that is currently handling the event. Once the
        *     event propagates through the scope hierarchy, this property is set to null.
        *   - `name` - `{string}`: name of the event.
-       *   - `stopPropagation` - `{function=}`: calling `stopPropagation` function will cancel
+       *   - `stopPropagation` - `{function}`: calling `stopPropagation` function will cancel
```

Comment:
The `=` was intentional to indicate that `stopPropagation` is not always available. It only exists on `$emit`ed events (not on `$broadcast`ed ones).